torch/_refs/__init__.py

```diff
@@ -2998,13 +3012,60 @@ def new_empty(
     )
 
 
-# TODO: missing kwargs (e.g. layout)
+@register_decomposition(torch.ops.aten.new_zeros)
+def new_zeros(
+    a: TensorLikeType,
+    size: ShapeType,
+    *,
+    dtype: Optional[torch.dtype] = None,
+    layout: Optional[torch.layout] = None,
+    device: Optional[torch.device] = None,
+    pin_memory: bool = False,
+) -> TensorLikeType:
+    r = a.new_empty(size, dtype=dtype, layout=layout, device=device, pin_memory=pin_memory)
```

Comment:
is it worth having `new_full` as a prim operator which would cover new_zeros/new_ones/new_full without requiring two operators & mutation

Reply:
Not sure but I can always route these to `new_full`, at least...